src/base/preferences.cpp

```diff
@@ -567,6 +567,16 @@ void Preferences::setServerDomains(const QString &str)
     setValue("Preferences/WebUI/ServerDomains", str);
 }
 
+QString Preferences::getCorsDomain() const
+{
+    return value("Preferences/WebUI/CorsDomain", QChar('*')).toString();
```

Comment:
What is "Cors"?

Reply:
Cors stands for Cross-Origin Resource Sharing. It's a collection of HTTP headers that allows a web-page that isn't hosted under the same domain as the content it's trying to access, to access that content. (The most common header being: `Access-Control-Allow-Origin` which is the one the users are given access to here)

Say you have application.com, where an application that makes HTTP requests to a server hosted on api.com. 

if it isn't explicitly stated by api.com that "pages hosted on application.com can access this content", a request made from application.com by javascript in a web application won't be forwarded to api.com.

You can read more about it here: https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS

Reply:
I can setup a small demo, if you'd like? :)

Reply:
Just please use this abbreviation in valid notation (i.e. CORS, all capital letters) whenever possible (in variable/method names, configuration file etc.) to avoid confusion with the regular word.